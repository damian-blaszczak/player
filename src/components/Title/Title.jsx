
import styled from 'styled-components';

const Title = styled.h2`
  font-size: 2rem;
  color: white;
  font-weight: 500;
  width: 100%;
  margin-left: 20px;
`;

export default Title;
