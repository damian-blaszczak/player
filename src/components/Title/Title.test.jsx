import React from 'react';
import { render } from '@testing-library/react';
import Title from './Title';

describe('Title', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Title />
    );
    expect(container).toMatchSnapshot();
  });

  it('should have white color', () => {
    const { container } = render(
        <Title />
    );

    const titleElement = container.querySelector('h2');
    expect(titleElement).toHaveStyle('color: white');
});
});
