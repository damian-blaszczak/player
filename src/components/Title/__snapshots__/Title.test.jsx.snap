// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<div>
  <h2
    class="sc-AxjAm ijAwJO"
  />
</div>
`;
